### PR TITLE
more doxygen

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #154 

Filling in some documentation.

### CODE REVIEW

**Comment by edwardhartnett** [comment]
This cannot pass CI until #153 is merged.

File: src/gdswzd_c.F90 [hunk 1]
```diff
@@ -166,7 +166,7 @@ module gdswzd_c_mod
   !> @param[out] xlat (npts) dx/dlat in 1/degrees
   !> @param[out] ylon (npts) dy/dlon in 1/degrees
   !> @param[out] ylat (npts) dy/dlat in 1/degrees
-  !> @param[out] AREA (npts) area weights in m**2 (Proportional to the
+  !> @param[out] AREA (npts) area weights in m**2 (Proportional to the ???)
```
Comment (GeorgeGayno-NOAA) [inline src/gdswzd_c.F90]: I am not sure what the statement "Proportional to the" refers to. I would remove it.

Comment (edwardhartnett) [reply src/gdswzd_c.F90]: Actually I found it. It should be:

```
 * @param area (npts) area weights in m**2 (Proportional to the 
 * square of the map factor in the case of conformal projections.)

```

File: src/ip2lib_4.h [hunk 2]
```diff
@@ -1,33 +1,147 @@
...
+ * @param nret number of valid points computed (-1 if
+ * projection unrecognized)
+ * @param crot (npts) clockwise vector rotation cosines
+ * @param srot (npts) clockwise vector rotation sines
+ * (ugrid=crot*uearth-srot*vearth, vgrid=srot*uearth+crot*vearth)
+ * @param xlon (npts) dx/dlon in 1/degrees
+ * @param xlat (npts) dx/dlat in 1/degrees
+ * @param ylon (npts) dy/dlon in 1/degrees
+ * @param ylat (npts) dy/dlat in 1/degrees
+ * @param area (npts) area weights in m**2 (Proportional to the
```
Comment (GeorgeGayno-NOAA) [inline src/ip2lib_4.h:141]: Here is an explanation of what the area weights are proportional to.

Comment (edwardhartnett) [reply src/ip2lib_4.h:141]: Where? To what do you refer?

**Review by GeorgeGayno-NOAA — APPROVED** [review]

**Pull request merged**